### PR TITLE
Fix OS X Installer, Fix Typo

### PR DESCRIPTION
**Pull request by kurisubrooks**
Fixes the broken OS X installer (#325), also a typo

### CODE REVIEW

**Comment by KGan** [comment]
Doesn't work with new Discord update. ModuleUpdater2 isn't there anymore, Why not just prepend this to the top, or right after `'use strict';`

File: Installers/Node/index.js [hunk 1]
```diff
@@ -99,8 +99,10 @@ function install() {
                     console.log("Injecting index.js");
 
                     var data = fs.readFileSync(_discordPath + _index).toString().split("\n");
-                    data.splice(_importSplice, 0, 'var _betterDiscord = require(\'betterdiscord\');\n');
-                    data.splice(_functionCallSplice, 0, splice);
+                    data = data.join("\n");
+                    data = data.replace('var _ModuleUpdater2 = _interopRequireDefault(_ModuleUpdater);\n','var _ModuleUpdater2 = _interopRequireDefault(_ModuleUpdater);\n\nvar _betterDiscord = require(\'BetterDiscord\');\n var _betterDiscord2;');
```
Comment (KGan) [inline Installers/Node/index.js]: Doesn't work with new Discord update. ModuleUpdater2 isn't there anymore, Why not just prepend this to the top, or right after 'use strict';

Comment (kurisubrooks) [inline Installers/Node/index.js]: Jiiks dropped all support for the macOS version of BetterDiscord, so I think it's worth while for me to just close this pull request, until such a time that Jiiks cares about the community again.

**Comment by nullpixel1** [comment]
please don't do this to us

**Comment by kurisubrooks** [comment]
Unfortunately it's not my choice. @Jiiks seems to no longer maintain this repo, nor cares about macOS, and I hate working with other code bases too much to pull this back into shape. Sorry @nullpixel1 😞

**Comment by nullpixel1** [comment]
I found a working PR. What a twat.

**Comment by kurisubrooks** [comment]
Think I could get a link to that?

**Comment by nullpixel1** [comment]
https://github.com/Jiiks/BetterDiscordApp/pull/369 @kurisubrooks

**Pull request closed without merging**